src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -1451,7 +1455,16 @@ public void AddTimeSeriesNameToMetadata(DocumentsOperationContext ctx, string do
             if (doc == null)
                 return;
 
-            tsName = EnsureOriginalName(ctx, docId, tsName);
+            try
+            {
+                tsName = EnsureOriginalName(ctx, docId, tsName);
+            }
+            catch (Exception e)
+            {
+                var error = $"Unable to locate the original time-series '{tsName}' in document '{docId}'";
+                if (_logger.IsInfoEnabled)
```

Comment:
Is it okay to ignore this error, aside from the log?
What would be the behavior of the system?

Reply:
If I don't ignore this it might take down the replication or import.
And if we fail we will use the initial `tsName` that we got from the argument.
Worst-case as I see it, is that the metadata might contain wrong casing.